sdcm/cluster.py

```diff
@@ -2373,6 +2383,9 @@ def _service_cmd(self, service_name: str, cmd: str, timeout: int = 500, ignore_s
     def get_service_status(self, service_name: str, timeout: int = 500, ignore_status=False):
         return self._service_cmd(service_name=service_name, cmd='status', timeout=timeout, ignore_status=ignore_status)
 
+    def is_service_exists(self,  service_name: str, timeout: int = 500) -> bool:
+        return self._service_cmd(service_name=service_name, cmd='status', timeout=timeout, ignore_status=True).return_code == 0
```

Comment:
It won't work correctly - systemctl status can return !=0 in case service is not running or failed.
`systemctl list-unit-files | grep {service_name}.service` could work better

Reply:
another option would be to verify scylla version

Reply:
I can change the command, but doing so based on version complicates things more then needed

Reply:
change the command running it again:
- [ ] #33: https://jenkins.scylladb.com/job/scylla-staging/job/fruch/job/artifacts-rocky9-test/33/
- [ ] #49: https://jenkins.scylladb.com/job/scylla-staging/job/fruch/job/artifacts-ubuntu2204-test/49/